packages/jaeger-ui/src/utils/config/get-config.test.js

```diff
@@ -54,25 +55,31 @@ describe('getConfig()', () => {
     });
   });
 
-  describe('`window.getJaegerUiConfig` is a function', () => {
+  describe('index functions are injected by backend', () => {
     let embedded;
     let getJaegerUiConfig;
+    let capabilities;
+    let getJaegerStorageCapabilities;
 
     beforeEach(() => {
       getConfig.apply({}, []);
       embedded = {};
       getJaegerUiConfig = jest.fn(() => embedded);
       window.getJaegerUiConfig = getJaegerUiConfig;
+      capabilities = defaultConfig.storageCapabilities;
+      getJaegerStorageCapabilities = jest.fn(() => capabilities);
+      window.getJaegerStorageCapabilities = getJaegerStorageCapabilities;
```

Comment:
can we do this and not expose unnecessary vars in outer scope?
```suggestion
      window.getJaegerUiConfig = jest.fn(() => embedded);
      capabilities = defaultConfig.storageCapabilities;
      window.getJaegerStorageCapabilities = jest.fn(() => capabilities);
```

Reply:
Done